apps/hash-ai-worker-ts/src/activities/infer-entities/create-entities.ts

```diff
@@ -53,33 +76,100 @@ export const createEntities = async ({
         (proposedEntities ?? []).map(async (proposedEntity) => {
           const { properties = {} } = proposedEntity;
 
+          const nameProperties = ["name", "display-name", "preferred-name"];
+          const propertyKeysToMatchOn = typedKeys(properties).filter((key) =>
+            nameProperties.includes(
+              // capture the last part of the path, e.g. /@example/property-type/name/ -> name
+              key.match(/([^/]+)\/$/)?.[1] ?? "",
+            ),
+          );
```

Comment:
I don't think it's worth including the `title` as it'd involve fetching the property types as well (which we don't currently do) – not a big deal but since we know the slug and title are equivalent right now it seems somewhat pointless

Reply:
Sorry for the confusion, but I was referring to including `"title"` in the `nameProperties` array - not comparing the titles of the property types instead of the slug. Doesn't block merging though.